src/template.js

```diff
@@ -305,6 +305,41 @@ export class Templates {
       resolver(templateClass);
     }
   }
+
+  /**
+   * @param {!BaseTemplate} impl
+   * @param {!JSONObject} data
+   * @private
+   */
+  render_(impl, data) {
+    const root = impl.render(data);
+    const anchors = root.getElementsByTagName('a');
+    for (let i = 0; i < anchors.length; i++) {
+      const anchor = anchors[i];
+      if (!anchor.hasAttribute('href')) {
+        // Ignore anchors without href.
+        continue;
+      }
+
+      const target = anchor.getAttribute('target');
+      let updatedTarget;
+      if (target == '_top' || target == '_blank') {
+        // Allow _top and _blank.
+        updatedTarget = target;
+      } else if (!target) {
+        // TODO(dvoytenko, #1572): Change this once sanitization of targets is
+        // addressed. The most correct value is probably _top.
+        updatedTarget = '_blank';
```

Comment:
I think you have these conditionals confused in the comment: anything without a `target` should behave as `target="_top"` and anything with should temporarily be assigned to `target="_blank"`.

PS. can you swap the conditionals to avoid the `!target`-`else` double negation?

```js
if (target == '_top' || target == '_blank') {
  //...
} else if (target) {
  // ...
} else {
  // ...
}